code/modules/mob/living/carbon/human/species_types.dm

```diff
@@ -75,6 +75,13 @@
 
 	return randname
 
+/datum/species/lizard/on_species_gain(mob/living/carbon/C)
+	if(C.dna.blood_type != "L")
+		C.dna.blood_type = "L"
```

Comment:
This could possibly be a generic species var ? Not that i expect we'll be adding many species with unique bloodtypes any time soon.